boot/boot_serial/test/src/testcases/boot_serial_img_msg.c

```diff
@@ -35,7 +35,7 @@ TEST_CASE(boot_serial_img_msg)
      * 00000018  63 6c 65 6e 1a 00 01 14  |clen....|
      * 00000020  e8 63 6f 66 66 00        |.coff.|
      */
-    static const uint8_t payload[] = { 
```

Comment:
Although it is fine to fix whitespace errors like this, it should be done in a separate commit.

Reply:
I am going to transfer such modifications into a separate commit.